aspnetcore/blazor/tutorials/movie-database-app/part-1.md

```diff
@@ -0,0 +1,435 @@
+
+:::zone-end
+
+:::zone pivot="cli"
+
+Confirm that you have the latest [.NET SDK](https://dotnet.microsoft.com/download/dotnet) installed.
+
+In a command shell:
+
+* Change to the directory using the `cd` command to where you want to create the project folder (for example, `cd c:/users/Bernie_Kopell/Documents`).
+* Use the [`dotnet new` command](/dotnet/core/tools/dotnet-new) with the [`blazor` project template](/dotnet/core/tools/dotnet-new-sdk-templates#blazor) to create a new Blazor Web App project. The [`-o|--output` option](/dotnet/core/tools/dotnet-new#options) passed to the command creates the project in a new folder named `BlazorWebAppMovies` at the current shell directory location.
+
+  > [!IMPORTANT]
+  > Name the project `BlazorWebAppMovies`, including matching the capitalization, so the namespaces match for code that you copy from the tutorial to the app as you follow the tutorial.
+
+  ```dotnetcli
+  dotnet new blazor -o BlazorWebAppMovies
+  ```
+
+:::zone-end
+
+## Run the app
+
+:::zone pivot="vs"
+
+Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> on the keyboard to run the app without the debugger.
+
+Visual Studio displays the following dialog when a project isn't configured to use SSL:
+
+![Trust self-signed certificate dialog](~/blazor/tutorials/movie-database-app/part-1/_static/trust-certificate.png)
+
+Select **Yes** if you trust the ASP.NET Core SSL certificate.
+
+The following dialog is displayed:
+
+![Security warning dialog](~/blazor/tutorials/movie-database-app/part-1/_static/install-certificate.png)
+
+Select **Yes** to acknowledge the risk and install the certificate.
+
+For information on trusting the development certificate for the Firefox browser, see <xref:security/enforcing-ssl#trust-the-https-certificate-with-firefox-to-prevent-sec_error_inadequate_key_usage-error>.
+
+Visual Studio:
+
+* Compiles and runs the app.
+* Launches the default browser at `https://localhost:{PORT}`, which displays the app's UI. The `{PORT}` placeholder is the random port assigned to the app when the app is created. If you need to change the port due to a local port conflict, you may do so in the project's `Properties/launchSettings.json` file.
+
+Navigate the pages of the app to confirm that the app is working normally.
+
+:::zone-end
+
+:::zone pivot="vsc"
+
+For information on trusting the HTTPS certificate for browsers other than Firefox, see the [HTTPS development certificate trust guidance](xref:security/enforcing-ssl#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos). When using the Firefox browser, see the [certificate trust guidance for Firefox](xref:security/enforcing-ssl#trust-the-https-certificate-with-firefox-to-prevent-sec_error_inadequate_key_usage-error).
+
```

Comment:
Ok, but the reason that Rick added this to the MVC/RP tutorials was based on reader feedback where devs opened one or more issues on the subject. This might be the only way to get newbies to stop opening issues on it. I think we can try to ward off such issues by at least having these cross-links in the additional resources at the bottom of this article. I'll give that a shot. Still tho, I'm concerned about it 😨.